routing/routing_session.cpp

```diff
@@ -786,6 +809,29 @@ void RoutingSession::CopyTraffic(traffic::AllMwmTrafficInfo & trafficColoring) c
   TrafficCache::CopyTraffic(trafficColoring);
 }
 
+void RoutingSession::ProcessCameraWarning()
```

Comment:
Do not forget about `CHECK_THREAD_CHECKER(m_threadChecker, ());` when you add method to RoutingSession

Reply:
done